.github/workflows/ci.yml

```diff
@@ -3,11 +3,11 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     name: Erlang/OTP ${{matrix.otp}}
     strategy:
       matrix:
-        otp: ['23', '24', '25']
+        otp: ['24', '25', '26']
```

Comment:
If we enable Erlang 27 compatibility in this PR, would it make sense to test on Erlang 27 and not stop at 26?

Reply:
Yes, my bad, amended the last commit. Thanks!